schemas/json/block.json

```diff
@@ -336,6 +336,59 @@
 					"description": "By default, all blocks will appear in the inserter, block transforms menu, Style Book, etc. To hide a block from all parts of the user interface so that it can only be inserted programmatically, set inserter to false.",
 					"default": true
 				},
+				"layout": {
+					"type": [ "boolean", "object" ],
```

Comment:
Sooo we have a bit of a problem here, in that our schema validation doesn't allow properties to have multiple types like this 😂 

Cc @ajlende who added strict validation in #44423.

Reply:
Could we use `"oneOf"`? Something like `"settingsPropertiesTypography.fluid"` has:

```json
"oneOf": [
	{ "type": "boolean" },
	{ "type": "object" }
]
```

Reply:
ooooooh

Reply:
updated; still seeing test failures locally though

Reply:
seems to have fixed the issue on CI! yay 🎉

Reply:
Sweeeeeet! 🌮